src/main/java/io/reactivex/internal/fuseable/ConditionalSubscriber.java

```diff
@@ -16,7 +16,7 @@
 import io.reactivex.FlowableSubscriber;
 
 /**
- * A Subscriber with an additional onNextIf(T) method that
+ * A Subscriber with an additional {@link #tryOnNext(Object)} method that
  * tells the caller the specified value has been accepted or
  * not.
  *
```

Comment:
separate topic, but when reading this I was wondering if we should call out pattern/antipattern for calling `tryOnNext` *and also* `onNext`. Is there a case where this should happen? If so, or not, would that description end up in javadoc or somewhere else?